how-to-build.md

```diff
@@ -18,30 +18,75 @@ each of our major platforms:
     sudo apt-get install build-essential
     sudo apt-get install libproc-dev
     sudo apt-get install libssl-dev
+    # To install cmake see Building CMake from Source below.
 
 ### Linux Mint 15, 16, Ubuntu 13.04, 13.10
 
     sudo apt-get install build-essential
     sudo apt-get install libprocps0-dev
     sudo apt-get install libssl-dev
+    # To install cmake see Building CMake from Source below.
 
 ### Linux Mint 17, 17.1, Ubuntu 14.04
 
     sudo apt-get install build-essential
     sudo apt-get install libprocps3-dev
     sudo apt-get install libssl-dev
+    # To install cmake see Building CMake from Source below.
 
 ### Fedora 17, 18, 19, 20, Amazon Linux 2012.09
 
     sudo yum install gcc gcc-c++
     sudo yum install procps-devel
+    # To install cmake see Building CMake from Source below.
 
 ### OS X 10.10 and 10.11
 
 On OS X, Clang is used as the C/C++ compiler by default. Clang is installed
 as part of Xcode. Xcode 7.0 or newer is required, and can be installed via
 the Mac App Store.
 
+Setting up a development environment can convienently be achieved using a
+package manager called brew. See https://brew.sh for install instructions.
+
+    brew install cmake
+
+### Windows
+
+On Windows, navigate to the following websites in your browser
+to download the appropriate installers.
+
+- Visual Studio 2017: https://www.visualstudio.com/
+- CMake version 3.6 or higher: https://cmake.org/download/
+
+### Building CMake from Source
```

Comment:
I like the idea of the shell script, that seems much simple and dependable. Thanks!